spec/rubocop/cli_spec.rb

```diff
@@ -11,7 +11,7 @@
     it 'returns 1' do
       allow_any_instance_of(RuboCop::Runner)
         .to receive(:aborting?).and_return(true)
-      create_file('example.rb', '# encoding: utf-8')
+      create_file('example.rb', '')
```

Comment:
I think you should tweak this helper to creating empty files when passed just a file name.

Reply:
I was about to add a `''` default value to the `content` parameter of this method, but actually... Maybe extracting a separate `create_empty_file(name)` method that calls `create_file(name, '')` is slightly more clear?